presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorColumnHandle.java

```diff
@@ -114,6 +118,11 @@ public boolean isShardUuid()
         return isShardUuidColumn(columnId);
     }
 
+    public boolean isBucketNumber()
```

Comment:
This method is not used. Also, `isShardUuid()` is no longer used, so both can be removed.